src/Main.java

```diff
@@ -0,0 +1,585 @@
+import java.util.*;
+import java.io.*;
+/**
+ * Main run method that takes your order and outputs the subtotal, tax, and total.
+ * @author N. Wan.
+*/
+public class Main {
+
+    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+    static StringTokenizer st; 
+    public static void main(String[] args) throws IOException{
+
+        //ArrayList<String> foodType = new ArrayList<String>();
+
+        // Declare variables
+        int itemNum;
+        boolean going = true; 
+        char addToCart;
+        boolean isCrispy = true;
+        int spice = 0;
+        int buns = 0; 
+        int beef = 1; 
+        boolean isBeyondMeat = true;
+        boolean isLettuceBun = true;
+        boolean isBagelBun = true;
+        boolean isSausageMeat = true; 
+        char yOrN;
+        int numOption;
+        Order order = new Order();
+        String size = "Small";
+
+        while(going == true){
+            // Menu
+            System.out.println("-------------------");
+            System.out.println("BURGERS");
+            System.out.println("1 | Morning Burger");
+            System.out.println("2 | Stacked Burger");
+            System.out.println("3 | Vegan Burger");
+            System.out.println("4 | Chicken Burger");
+            System.out.println("SIDES");
+            System.out.println("5 | Fries");
+            System.out.println("6 | Waffle Fries"); 
+            System.out.println("7 | Poutine");
+            System.out.println("Drinks");
+            System.out.println("8 | Fruitopia");
+            System.out.println("9 | Dr. Pepper"); 
+            System.out.println("10| Coca-Cola");
+            System.out.println("11| Root Beer");
+            System.out.println("-------------------");
+
+            System.out.print("Enter the item you would like to view: ");
+            itemNum = readInt();
+
+        
+            if(itemNum > 0 && itemNum < 5){
+
+                // Interface for Morning Burger
+                if (itemNum == 1) {
+                    System.out.println("Morning Burger: ");
+                    System.out.println("- Toppings: Cheese, Egg, Mayonnaise, Hash brown, bacon" );
+                    System.out.println("- Choice of either bagel bun or potato bun");
+                    System.out.println("- Choice of either sausge meat or beef meat");
+                    System.out.println("Price: $5.99");
```

Comment:
the printout of food options for the menu should be encapsulated within the class.  By hardcoding it into the Main program, any changes to the food class definition (i.e MorningBurger price change) would require an edit in the Main and food subclass, breaking encapsulation principles